llama_stack/providers/tests/agents/fixtures.py

```diff
@@ -55,7 +61,28 @@ def agents_meta_reference() -> ProviderFixture:
     )
 
 
+@pytest.fixture(scope="session")
+def tool_runtime_memory() -> ProviderFixture:
+    return ProviderFixture(
+        providers=[
+            Provider(
+                provider_id="memory-runtime",
+                provider_type="inline::memory-runtime",
+                config={},
+            ),
+            Provider(
+                provider_id="brave-search",
+                provider_type="inline::brave-search",
+                config={
+                    "api_key": os.environ["BRAVE_SEARCH_API_KEY"],
```

Comment:
should this be `get_env_or_fail`?

Reply:
no we skip the test if env is not present